src/panels/config/zwave/zwave-node-config.ts

```diff
@@ -0,0 +1,388 @@
+                @iron-select=${this._selectedConfigParameterChanged}
+              >
+                ${this.config.map(
+                  (state) => html`
+                    <paper-item>
+                      ${state.key}: ${state.value.label}
+                    </paper-item>
+                  `
+                )}
+              </paper-listbox>
+            </paper-dropdown-menu>
+          </div>
+          ${this._configItem
+            ? html`
+                ${this._configItem.value.type === "List"
+                  ? html`
+                      <div class="device-picker">
+                        <paper-dropdown-menu
+                          .label=${this.hass!.localize(
+                            "ui.panel.config.zwave.node_config.config_value"
+                          )}
+                          dynamic-align
+                          class="flex"
+                          .placeholder=${this._configItem.value.data}
+                        >
+                          <paper-listbox
+                            slot="dropdown-content"
+                            .selected=${this._configItem.value.data}
+                            @iron-select=${this._configValueSelectChanged}
+                          >
+                            ${this._configItem.value.data_items.map(
+                              (state) => html`
+                                <paper-item>${state}</paper-item>
+                              `
+                            )}
+                          </paper-listbox>
+                        </paper-dropdown-menu>
+                      </div>
+                    `
+                  : ""}
+                ${["Byte", "Short", "Int"].includes(this._configItem.value.type)
+                  ? html`
+                      <div class="card-actions">
+                        <paper-input
+                          .label=${this._configItem.value.data_items}
+                          type="number"
+                          .value=${this._configItem.value.data}
+                          .max=${this._configItem.value.max}
+                          .min=${this._configItem.value.min}
+                          @value-changed=${this._configValueInputChanged}
+                        >
+                        </paper-input>
+                      </div>
+                    `
+                  : ""}
+                ${["Bool", "Button"].includes(this._configItem.value.type)
+                  ? html`
+                      <div class="device-picker">
+                        <paper-dropdown-menu
+                          .label=${this.hass!.localize(
+                            "ui.panel.config.zwave.node_config.config_value"
+                          )}
+                          class="flex"
+                          dynamic-align
+                          .placeholder=${this._configItem.value.data}
+                        >
+                          <paper-listbox
```

Comment:
Why not use a checkbox?

Reply:
It was a dropdown originally so I just left it like that.